scene/audio/audio_stream_player.h

```diff
@@ -57,7 +57,7 @@ class AudioStreamPlayer : public Node {
 	void _set_playing(bool p_enable);
 	bool _is_active() const;
 
-	Vector<AudioFrame> _get_volume_vector();
+	LocalVector<AudioFrame> _get_volume_vector();
```

Comment:
Since this is public it shouldn't use `LocalVector` IMO, as discussed

Reply:
Though, it's a private method. 😕

Reply:
My bad that's true